File: arm.64/langs/java/src/trip.java
package pat;

import java.util.Arrays;
import java.util.List;

public class trip {
    public static List<type.Triplet> trip() {
        return Arrays.asList(
            new type.Triplet("/version", "{}", "{\"once\":true}"),
            new type.Triplet("/storage", "{}", "{\"once\":true}"),
            new type.Triplet("sql.help", "{}", "{\"once\":true}")
        );
    }
}

File: arm.64/langs/python/src/trip.py
from typing import List
from type import Triplet

def trip() -> List[Triplet]:
    return [
        Triplet(address="/version", payload="{}", options='{"once":true}'),
        Triplet(address="/storage", payload="{}", options='{"once":true}'),
        Triplet(address="sql.help", payload="{}", options='{"once":true}')
    ]
